test_api/04_test_updating_resources.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+
+from test_api.constant import TestData
+
+
+@pytest.mark.parametrize('todos_id', [TestData.TODOS_VALUE, (TestData.TODOS_VALUE - 1), (TestData.TODOS_VALUE - 50)])
+def test_update_resources_put_positive(create_session, workplace, todos_id):
+    response = create_session.put(url=f'{workplace}/{todos_id - 1}', json=TestData.DICT_UNIT_1)
+
+    assert response.status_code == 200
+    j_res = response.json()
+    assert j_res['title'] == TestData.TITLE
+    assert j_res['completed'] is False
```

Comment:
You can add **userId** field check and use **TestData.DICT_UNIT_1** values for data consistency:

```
    assert j_res['userId'] == TestData.DICT_UNIT_1['userId']
    assert j_res['title'] == TestData.DICT_UNIT_1['title']
    assert j_res['completed'] is TestData.DICT_UNIT_1['completed']
```

Reply:
added